tests/logictest/main.py

```diff
@@ -7,17 +7,25 @@
 
 from config import mysql_config, http_config
 
-if __name__ == '__main__':
-    disable_mysql_test = os.getenv("DISABLE_MYSQL_LOGIC_TEST")
-    if disable_mysql_test is None:
-        mySQL = TestMySQL("mysql")
-        mySQL.set_driver(mysql_config)
-        mySQL.set_label("mysql")
-        mySQL.run_sql_suite()
-
-    disable_http_test = os.getenv("DISABLE_HTTP_LOGIC_TEST")
-    if disable_http_test is None:
-        http = TestHttp("http")
+import fire
+
+
+def run(pattern=".*", enable_mysql=True, enable_http=True):
+    """
+    Run tests
+    """
+    if enable_mysql:
```

Comment:
I think we still need to push down the feature flags by some flags or environment variables

Reply:
You can use config.py to keep default value, make sure always use the config of environment variables. All current environment variables must keep.

Reply:
Ok, I just restore the env variables.